internal/nodes/avi_vrf_translator.go

```diff
@@ -235,8 +256,8 @@ func (o *AviObjectGraph) addRouteForNode(node *v1.Node, vrfName string, routeid
 		}
 
 		nodeRoutes = append(nodeRoutes, &nodeRoute)
+		routeIdList[routeid] = struct{}{}
 		routeid++
```

Comment:
is this increment required if we are finding free route linearly?

Reply:
@aaha97: If IPV4 and IPV6 PODCIDRs are present for given node, then we need to add those two routes for given node and this `routeid` increment is required to take care of that IPV4 and IPV6 addresses.